api/swagger-spec/v1.json

```diff
@@ -13193,7 +13193,7 @@
       "type": "versioned.Event",
       "method": "GET",
       "summary": "watch individual changes to a list of Secret",
-      "nickname": "watchNamespacedSecretList",
+      "nickname": "watchNamespacedSecretListForAllNamespaces",
```

Comment:
Removing first "Namespaced" make more sense to me. At some point I thought about it, but I am not sure why I didn't go with it.